### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by ijemmao**
* Increased the width of the nav bar
* Updated hover effect due to Young's request
* Included each member's year

### CODE REVIEW

File: src/stylesheets/components/_user_card.scss [hunk 1]
```diff
@@ -12,10 +12,11 @@
   border-radius: 5px;
   box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.1);
   cursor: pointer;
-  transition: bottom .2s ease-in-out;
+  transition: all .2s ease-in-out;
 
   &:hover {
     bottom: 10px;
+    box-shadow: 0px 15px 40px rgba(0, 0, 0, 0.2);
```
Comment (randaline11) [inline src/stylesheets/components/_user_card.scss]: another case... maybe predefine shadow?

**Comment by randaline11** [comment]
screen shot 2018-10-11 at 11 24 29 am
So I'm noticing that some people have years and no names (Ex. Sarah Hong) and some people have names and no years (ex. Abby Starr). I'm assuming this is just a gap in our data?
Also, when someone doesn't have a profile picture, can we put in a nicer-looking icon?

**Review by randaline11 — DISMISSED** [review]

File: src/stylesheets/components/_user_card.scss (hunk 1 above)
Comment (ijemmao) [reply src/stylesheets/components/_user_card.scss]: Yeah, that's a good point. I'm gonna create `$shadow` and $shadow-hover` variables.

**Comment by ijemmao** [comment]
@randaline11 Yeah, it's definitely a gap in our data. I'll add filler user image if they don't have one.

**Comment by ijemmao** [comment]
@johnlev How are images getting populated in the database? It seems like there are direct links to Imgur image pages, but not raw images hosted on the Imgur site. Is there a way we can clean this up?

**Comment by johnlev** [comment]
>@johnlev How are images getting populated in the database? It seems like there are direct links to Imgur image pages, but not raw images hosted on the Imgur site. Is there a way we can clean this up?

@ijemmao yeah, the photos but was always weird but was never a priority for fixing. It should be populated by google account or personnel files. At some point we should be looking into storing those in some s3 database or something

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ijemmao** [comment]
> > @johnlev How are images getting populated in the database? It seems like there are direct links to Imgur image pages, but not raw images hosted on the Imgur site. Is there a way we can clean this up?
> 
> @ijemmao yeah, the photos but was always weird but was never a priority for fixing. It should be populated by google account or personnel files. At some point we should be looking into storing those in some s3 database or something
> 
> Sent with [GitHawk](http://githawk.com)

@johnlev That makes sense. I'm wondering if we could alter the backend so that it only takes in url paths that end with `.jpg` or `.png` and reject others.

The issue that I'm dealing with right now is that the `photoUrl` property holds a url that points to website, and not necessarily an image. I don't think I can check if I actually pulled in an image or not. 

Any ideas?

**Review by randaline11 — APPROVED** [review]

**Pull request merged**